pkg/storage/stores/shipper/storage/client.go

```diff
@@ -14,11 +14,17 @@ const delimiter = "/"
 
 // Client is used to manage boltdb index files in object storage, when using boltdb-shipper.
 type Client interface {
-	ListTables(ctx context.Context) ([]string, error)
-	ListFiles(ctx context.Context, tableName string) ([]IndexFile, error)
+	ListFiles(ctx context.Context, tableName string) ([]IndexFile, []string, error)
```

Comment:
Could you may be compose this interface with the two interface at the end, and also move then at the top ?

Reply:
Hah, that was the plan and I missed doing that. Thanks for pointing it out!